File: testcases/paper/Methods/Methods.cpp
#include <iostream>

// Outside method
void outsideMethod() {
    std::cout << "Outside method called.\n";
}

// Class definition
class MyClass {
public:
    void methodOne() {
        std::cout << "Method one called.\n";
    }

    void methodTwo() {
        std::cout << "Method two called.\n";
        methodOne(); // Calls methodOne
    }

    void methodThree() {
        std::cout << "Method three called.\n";
        outsideMethod(); // Calls outside method
    }
};

File: testcases/paper/Methods/Methods.py
def outsideMethod():
    print("Outside method called.")

class MyClass:
    def methodOne(self):
        print("Method one called.")

    def methodTwo(self):
        print("Method two called.")
        self.methodOne() # Calls methodOne

    def methodThree(self):
        print("Method three called.")
        outsideMethod() # Calls outside method
